source/CardGames.py

```diff
@@ -396,16 +396,18 @@ def check_illegal_card(self):
         previous_hideout = self.cards_in_play[-1]
       hideouts = input("Select two viable cards you want to place as hideouts separated only by a comma (1,2,3...): ").split(',')
 
-      number_of_cards_placed = len(hideouts)
-      idx = 0
+      
+
+      if len(hideouts) != 2:
+        print("Select two inputs!")
       fugitive_card_values = []
 
       #pull value of owned cards and add them to list
       for fug_card in self.fugitive_deck:
         fugitive_card_values.append(fug_card.value)
 
       #for number of hideouts placed
-      while idx < number_of_cards_placed:
+      for idx in range(len(hideouts)):
         #convert input string to int
         card = int(hideouts[idx])
         #check if the hideout placed is a card the player owns
```

Comment:
On line 417, where you loop through self.fugitive_deck, it might be better to loop through the fugitive_card_values list you made earlier if your goal is to obtain the values again.

Reply:
Hey brother, good catch. Made that change as well. Makes the code a bit cleaner!